quirc/examples/esp-eye-qrcode/main/include/app_peripherals.h

```diff
@@ -0,0 +1,248 @@
+#pragma once
+
+#include "freertos/FreeRTOS.h"
+#include "freertos/queue.h"
+#include "freertos/task.h"
+#include "freertos/semphr.h"
+
+#include "esp_camera.h"
+
+#define CONFIG_CAMERA_MODULE_ESP_EYE 1
```

Comment:
I would recommend bringing board definitions using the board support package. This way to support a new board, work needs to be done only on BSP side, and the example doesn't have to be updated.

Example: https://github.com/espressif/qrcode-demo/blob/188cc80ca4e3654f597f00cdcb2193fad1709010/main/idf_component.yml#L4

(It's okay if that means that fewer boards are supported. After all, it's just an example.)

Reply:
Thank you Ivan for your kind and detailed feedback. 

Using the bsp approach would be the best choice for sure. Right now though I don't see the bsp for the esp-eye (meaning the [older one](https://www.espressif.com/en/products/devkits/esp-eye/overview)). I could setup a very basic bsp for this board starting form the esp32-s3-eye and update this example accordingly. 

I will receive and Esp32-s3-eye the next week to test this example on it anyway.

Reply:
Ah yes, we don't seem to have a BSP for the original ESP-EYE.

I think at this point it's probably safe to recommend ESP32-S3-EYE for new applications, so I wouldn't mind keeping this example supported on that board only. Later when we add ESP-EYE BSP, we can update the example to support it as well.

Reply:
Ok, then it will take a few days to receive the hardware and make the changes. 
If it's better, I can close this PR here and reopen when everything it's in place. Otherwise I'll leave it here and update it in a few days.

Reply:
You can keep this PR, just make the changes and push to the same branch when you are ready. Later once everything is ready you can clean up the commit history.